CHANGELOG.md

```diff
@@ -21,13 +21,15 @@
     - `sdk.setFromUri`
     - `sdk.getFromUri`
     - `sdk.removeFromUri`
+    - `sdk.handlePostLoginRedirect`
```

Comment:
I think we can name this `handleLoginRedirect` which matches better with `isLoginRedirect`. Also is `isLoginRedirect` being hoisted out of the `token` namespace? Either `isLoginRedirect` should be hoisted up, or `handleLoginRedirect` should be put inside the `token` namespace, because these 2 methods belong together.